storage/cloud.py

```
@@ -157,17 +161,80 @@ def __init__(
+
+    def create_trace(self, operation_name, params, **kwargs):
+        try:
+            # ctx = get_tracecontext()
+            name = "boto3." + str(operation_name)
+            with tracer.start_as_current_span(name) as span:
+                for k in params:
+                    span.set_attribute(k, str(params[k]))
+                for k in kwargs:
+                    span.set_attribute(k, str(kwargs[k]))
+                span.set_status(StatusCode.OK)
+        except Exception as trerr:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)